packages/server/src/routes/members/index.ts

```diff
@@ -21,7 +21,7 @@ export const listMembers = (membersCollection: CollectionReference) =>
         id: member.id,
         created_at: member.get("created_at"),
         full_name: member.get("full_name"),
-        identity_video_url: member.get("identity_video_url"),
+        identityVideoReference: member.get("identityVideoReference"),
```

Comment:
this PR is closed since it's not backwards compatible, but i think it's lame we even have any inconsistency about using snake_case here at all so I wanted to incrementally move new stuff to camel case. But given that we'll have to support old versions indefinitely, i'll just make it be snake_case and embrace the pointless inconsistency

Reply:
I thought in general we did underlines for field names

Reply:
the only place we ever do it is operation names, probably bc @rahulgi defined the schema when he was more in python mode than javascript, so every time we go from operations to code we have to keep renaming things from snake case to camel case; it's pointless but we've committed

Reply:
can we just migrate things? i'm ok with doing camelcase moving forward